docs/user/admin/settings.rst

```diff
@@ -824,3 +824,55 @@ To Re-enable Data Sources:::
       }
     }
 
+plugins.query.field_type_tolerance
+==================================
+
+Description
+-----------
+
+This setting controls whether preserve arrays. If this setting is set to false, then an array is reduced
```

Comment:
highlight default value is True.

Reply:
Reworked the documentation. It should be more clear what the default is.